packages/@aws-cdk/aws-apigateway/lib/rate-limited-api-key.ts

```diff
@@ -0,0 +1,132 @@
+import { Construct, IResource as IResourceBase, Resource } from '@aws-cdk/core';
+import { ApiKey } from './api-key';
+import { ResourceOptions } from "./resource";
+import { RestApi } from './restapi';
+import { QuotaSettings, ThrottleSettings, UsagePlan, UsagePlanPerApiStage } from './usage-plan';
+
+/**
+ * API keys are alphanumeric string values that you distribute to
+ * app developer customers to grant access to your API
+ */
+export interface IRateLimitedApiKey extends IResourceBase {
+  /**
+   * The API key ID.
+   * @attribute
+   */
+  readonly keyId: string;
+}
```

Comment:
This interface is not needed. Use `IApiKey` in its place.

Reply:
Done!